app/controllers/projects_controller.rb

```diff
@@ -84,6 +85,7 @@ def index
                      current_power.projects
                    end
 
+        projects = projects.where(id: params[:projectId]) if project_id
```

Comment:
nit you probably meant to use the project_id var inside the clause as well

Reply:
done